src/ui/app.rs

```diff
@@ -282,11 +284,26 @@ mod inspector {
                             ObjectType::Port => {
                                 if let Some(parent) = global_borrow.parent_id() {
                                     let name = global_borrow.name().cloned().unwrap_or_default();
+                                    let format_dsp = global_borrow.props().get("format.dsp");
+                                    let media_type = if let Some(format_dsp) = format_dsp {
+                                        Some(if format_dsp.ends_with("audio") {
+                                            MediaType::Audio
+                                        } else if format_dsp.ends_with("midi") {
```

Comment:
`UMP` is another possible suffix that indicates MIDI
https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/5c5075f4d610af0efd2539ba0417f76afe492a65/src/examples/midi-src.c#L216